LocalMultiplayerAgent/SettingsJsonGenerator/scripts.js

```diff
@@ -2,6 +2,27 @@
 let BuildGuId = crypto.randomUUID();
 let SessionGuId = crypto.randomUUID()
 
+// These values match an enum for RunMode in the HTML file
+let RUN_MODE_WIN_PROCESS = "WinProcess";
```

Comment:
Maybe have all these be const?

Reply:
I think 'let' is const in js.

Reply:
No. It's only about scoping.
https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var
Basically let is more precisely similar to the way ALL C++/C#/etc languages work, where var is abusing the underlying language implementation in ways that "shouldn't technically work, but do".

Reply:
I am not familiar with JS, but what about this? https://www.tutorialspoint.com/const-vs-let-in-javascript

Reply:
I was wrong! Fixed.